Resources/Prototypes/Datasets/tips.yml

```diff
@@ -123,3 +123,48 @@
+  - "Vending machine food is bad for your health! Watch out for food trucks such as the Skipper or the McCargo - they can offer delicious food for a reasonable price!"
+  - "Meals cooked by chefs are more filling, more flavorful, and in some cases can even boost your movement speed for a short time!"
+  - "Each ID card can have a shuttle attached to it. If you need to purchase an additional shuttle, you can ask your local SR for another ID!"
+  - "The Station Representative can rename your shuttle! Usually it's enough to hand your ID and ask them to do it."
+  - "If your ship has a big crew, you can ask the SR for radio keys for your fellow crewmates. Some channels, such as Cargo or Science, are usually empty."
+  - "Before joining an ongoing round, you can see the list of ships registered on the New Frontiers. Perhaps some of them have jobs you would prefer?"
+  - "Don't forget to buy a medical implant before wandering into deep space! It will send a message to the local medics in the event of your death."
+  - "As a medic, you should cooperate with others when there's a medical emergency! It never helps when two medical ships arrive at the same emergency signal."
+  - "As a medic, you can ask the local SR for a medical PDA. It can double as a health analyzer!"
+  - "Medical ships often have access to the hypospray. An experienced chemist can turn it into a lethal weapon, or use it to safely put people to sleep."
+  - "If you're hurt, try to ask for medical help on the radio. Medics are often eager to heal the wounded, as their job usually gets boring after hours of waiting."
+  - "Always carry a GPS with yourself! It can help you locate your station, or tell your location to others and get saved."
+  - "Smuggling is very profitable, but also very risky! Remember to have an IFF console and hide your IFF before going for a dead drop."
+  - "As a smuggler, try to destroy any evidence of your crimes. Remember, your PDA has a notepad app, which can be used to store important notes in a digital form."
+  - "Security has a special IFF console that completely hides their ship on the radar! If you're an outlaw, watch out for those invisible Prowlers!"
+  - "Certain ships can act as motherships. They have shipyard consoles, which allow you to purchase and sell other smaller ships!"
+  - "As a crewmemeber, you can start a mutiny if your captain refuses to pay you. Be careful though, as the captain has the permission to execute the mutiny leader!"
+  - "Your PDA has a pre-installed news application. Check it out, perhaps the local reporter has posted some interesting news?"
+  - "There's a special app installed in your PDA, which allows you to see posted bounties. If you can fulfill one of them, you can ask the local SR or Sheriff for a reward!"
+  - "Remember, Frontier Outpost has a 200m wide safezone. It's forbidden to seriously harm people or property in any way within that limit!"
```

Comment:
Is this strictly true? Presently the rules (and word from admins) only restrict antag/griefing activity within 200m, but not all inter-character conflict full-stop.

Reply:
I'm not sure how to phrase it better, it could help if you gave a better example

Reply:
"Remember, Frontier Outpost has a 200m wide safe-zone. It's forbidden to engage in antagonist behavior or damage property in any way within that limit!"

Something like that?